<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QNoise</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="playerTab">
     <attribute name="icon">
      <iconset resource="myResources.qrc">
       <normaloff>:/myIcons/note</normaloff>:/myIcons/note</iconset>
     </attribute>
     <attribute name="title">
      <string>Player</string>
     </attribute>
     <widget class="QPushButton" name="addFilesButton">
      <property name="geometry">
       <rect>
        <x>549</x>
        <y>272</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="myResources.qrc">
        <normaloff>:/myIcons/add</normaloff>:/myIcons/add</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="previousButton">
      <property name="geometry">
       <rect>
        <x>119</x>
        <y>272</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="myResources.qrc">
        <normaloff>:/myIcons/previous</normaloff>:/myIcons/previous</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pauseButton">
      <property name="geometry">
       <rect>
        <x>168</x>
        <y>268</y>
        <width>48</width>
        <height>48</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="myResources.qrc">
        <normaloff>:/myIcons/pause</normaloff>:/myIcons/pause</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="playButton">
      <property name="geometry">
       <rect>
        <x>168</x>
        <y>268</y>
        <width>48</width>
        <height>48</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="myResources.qrc">
        <normaloff>:/myIcons/play</normaloff>:/myIcons/play</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="stopButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>225</x>
        <y>272</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="myResources.qrc">
        <normaloff>:/myIcons/stop</normaloff>:/myIcons/stop</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="nextButton">
      <property name="geometry">
       <rect>
        <x>274</x>
        <y>272</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="myResources.qrc">
        <normaloff>:/myIcons/next</normaloff>:/myIcons/next</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="clearButton">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>272</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="myResources.qrc">
        <normaloff>:/myIcons/clear</normaloff>:/myIcons/clear</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="musicTable">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>76</y>
        <width>571</width>
        <height>181</height>
       </rect>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>#</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Artist</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Title</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Album</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time</string>
       </property>
      </column>
     </widget>
     <widget class="Phonon::SeekSlider" name="mySeekSlider">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>45</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="statusLabel">
      <property name="geometry">
       <rect>
        <x>295</x>
        <y>10</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>QNoise</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="elapsedTimeLCD">
      <property name="geometry">
       <rect>
        <x>111</x>
        <y>46</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="remainingTimeLCD">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>46</y>
        <width>70</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="welcomeLabel">
      <property name="geometry">
       <rect>
        <x>176</x>
        <y>31</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Witaj w QNoise...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Phonon::VolumeSlider" name="myVolumeSlider">
      <property name="geometry">
       <rect>
        <x>347</x>
        <y>285</y>
        <width>125</width>
        <height>20</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="lovePlusButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>276</y>
        <width>32</width>
        <height>32</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="myResources.qrc">
        <normaloff>:/myIcons/love+</normaloff>:/myIcons/love+</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="loveMinusButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>276</y>
        <width>32</width>
        <height>32</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="myResources.qrc">
        <normaloff>:/myIcons/love-</normaloff>:/myIcons/love-</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="loveLabel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>12</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="myResources.qrc">:/myIcons/love</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="lyricsTab">
     <attribute name="icon">
      <iconset resource="myResources.qrc">
       <normaloff>:/myIcons/lyrics</normaloff>:/myIcons/lyrics</iconset>
     </attribute>
     <attribute name="title">
      <string>Lyrics</string>
     </attribute>
     <widget class="QTextBrowser" name="lyrics_textBrowser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>591</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="lastFmTab">
     <attribute name="icon">
      <iconset resource="myResources.qrc">
       <normaloff>:/myIcons/lastfm</normaloff>:/myIcons/lastfm</iconset>
     </attribute>
     <attribute name="title">
      <string>Last.fm</string>
     </attribute>
     <widget class="QTextBrowser" name="lastFm_textBrowser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>161</y>
        <width>591</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lastFm_sentLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Przesłanych : 0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lastFm_queuedLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>W kolejce : 0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lastFm_logoLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>96</width>
        <height>96</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="myResources.qrc">:/myIcons/lastfm</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="lastFm_updateButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>70</y>
        <width>83</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Odśwież</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="settingsTab">
     <attribute name="icon">
      <iconset resource="myResources.qrc">
       <normaloff>:/myIcons/settings</normaloff>:/myIcons/settings</iconset>
     </attribute>
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QLineEdit" name="settings_userLineEdit">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>55</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="settings_passwordLineEdit">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>85</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="settings_lastFmLogo">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>52</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="myResources.qrc">:/myIcons/lastfm</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="settings_lastFmUserTextLabel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Użytkownik</string>
      </property>
     </widget>
     <widget class="QLabel" name="settings_lastFmPassTextLabel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Hasło</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="settings_lastFmCheckBox">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>70</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Online</string>
      </property>
     </widget>
     <widget class="QLabel" name="settings_lyricsLogo">
      <property name="geometry">
       <rect>
        <x>58</x>
        <y>132</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="myResources.qrc">:/myIcons/lyrics</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="settings_lyricsCheckBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>153</y>
        <width>84</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Online</string>
      </property>
     </widget>
     <widget class="QLabel" name="settings_noteLogo">
      <property name="geometry">
       <rect>
        <x>52</x>
        <y>215</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="myResources.qrc">:/myIcons/note</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="settings_repeatCheckBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>233</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Powtarzaj</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="aboutTab">
     <attribute name="icon">
      <iconset resource="myResources.qrc">
       <normaloff>:/myIcons/info</normaloff>:/myIcons/info</iconset>
     </attribute>
     <attribute name="title">
      <string>About</string>
     </attribute>
     <widget class="QLabel" name="about_cutieLogoLabel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>128</width>
        <height>128</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="myResources.qrc">:/myIcons/cutielogo</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="about_cutieNameLabel">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>60</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>19</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>QNoise</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>100</y>
        <width>231</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Autorzy:

Patrick Żurowski
Dawid Dyderski
Michał Kołodziejczyk</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="about_versionLabel">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>220</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>wersja 0.9</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="about_licenseLabel">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>250</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Distributed under the GPL v2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QSlider</extends>
   <header>phonon_seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QSlider</extends>
   <header>phonon_volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="myResources.qrc"/>
 </resources>
 <connections/>
</ui>
